src/content/development/contribution-guide/_index.en.md

```diff
@@ -54,7 +54,13 @@ To be expanded
 
 ### Release Timeline
 
-To be expanded
+<!-- Source: https://docs.google.com/drawings/d/1wZCogcChCkX2PqoIuTx9I_iOgJuwsYjChVopq1nfqeM -->
+![Timeline Diagram](/images/timeline.png)
+
+Each release follows a fixed timeline, with 4 development sprints and one final sprint for stabilization.
```

Comment:
First sprint will be focused on epic design

Reply:
It is detailed in the sections, also "design" is part of the development. Please see #721